.circleci/config.yml

```diff
@@ -243,7 +243,7 @@ jobs:
       - prepare
       - run:
           name: TypeScript strict typecheck of individual subprojects
-          command: npm run typecheck -- --project client/landing/gutenboarding
+          command: npm run tsc -- --project client/landing/gutenboarding
```

Comment:
hmm are there other projects/directories that we should add here under `typecheck-strict`? it's currently just doing gutenboarding

Reply:
Yeah, that was discussed when I originally added this section. My reasoning was that this enables us to gradually add other strictly-typechecked parts of Calypso -- I'm doing that e.g. in #39373 (which, granted, is just an experiment).